engine/engine.go

```diff
@@ -52,13 +52,14 @@ var (
 // New starts a new engine
 func New() (*Engine, error) {
 	var b Engine
-	b.Config = &config.Cfg
+	b.Config = config.GetConfig()
```

Comment:
I think the purpose here is to always have a clean config to start with... the GetConfig() is risky as the returned config may already be used. So I think the previous was correct.
If you want to make the construction explicit, you can add a `config.New()` function and use that instead.